daemonset/nonrbac/fluentd.yaml

```diff
@@ -38,6 +41,33 @@ spec:
           readOnly: true
         - name: pos-files
           mountPath: /mnt/pos/
+        - name: fluentd-sumologic-config
```

Comment:
is there any reason not just mount whole directory other than individual files?

Reply:
Current Docker image seperates config across 2 directories.  `/fluentd/etc` and `fluentd/conf.d`.  So this was done this way to preserve and have a single configmap.  Other options would be:

- 2 config maps for each directory, then just have 2 volumens and mounts.
- Change entrypoint script and keep all config in single dir

Reply:
Made changes to simplify this...